server/api/lib/trees/regional_tree_permit.js

```diff
@@ -0,0 +1,233 @@
+const Log = require('../log');
+const fs = require('fs');
+const path = require('path');
+const xlsx = require('xlsx');
+const fetch = require('node-fetch');
+const moment = require('moment');
+const AbortController = require('abort-controller');
+const aws = require('aws-sdk');
+
+const TreePermit = require('../../model/tree_permit');
+const tpc = require('../../model/tree_permit_constants');
+const database = require('../../service/database');
+const Config = require('../../lib/config');
+
+// Regional tree permits were taken from here: 'https://www.moag.gov.il/yhidotmisrad/forest_commissioner/rishyonot_krita/Pages/default.aspx';
+const RAW_DATA_FOLDER = require('os').homedir() + '/raw_trees';
+const regionalTreePermitUrls = [
+	'https://www.moag.gov.il/yhidotmisrad/forest_commissioner/rishyonot_krita/Documents/Befor_galil_golan.XLS',
+	'https://www.moag.gov.il/yhidotmisrad/forest_commissioner/rishyonot_krita/Documents/after_galil_golan.XLS',
+
+	'https://www.moag.gov.il/yhidotmisrad/forest_commissioner/rishyonot_krita/Documents/Befor_amakim_galil_gilboa.XLS',
+	'https://www.moag.gov.il/yhidotmisrad/forest_commissioner/rishyonot_krita/Documents/after_amakim_galil_gilboa.XLS',
+
+	'https://www.moag.gov.il/yhidotmisrad/forest_commissioner/rishyonot_krita/Documents/befor_merkaz-sharon.XLS',
+	'https://www.moag.gov.il/yhidotmisrad/forest_commissioner/rishyonot_krita/Documents/after_merkaz-sharon.XLS',
+
+	'https://www.moag.gov.il/yhidotmisrad/forest_commissioner/rishyonot_krita/Documents/Befor_merkaz_shfela.XLS',
+	'https://www.moag.gov.il/yhidotmisrad/forest_commissioner/rishyonot_krita/Documents/after_merkaz_shfela.XLS',
+
+	'https://www.moag.gov.il/yhidotmisrad/forest_commissioner/rishyonot_krita/Documents/Befor_jerusalem.XLS',
+	'https://www.moag.gov.il/yhidotmisrad/forest_commissioner/rishyonot_krita/Documents/after_jerusalem.XLS',
+
+	'https://www.moag.gov.il/yhidotmisrad/forest_commissioner/rishyonot_krita/Documents/Befor_darom.XLS',
+	//'https://www.moag.gov.il/yhidotmisrad/forest_commissioner/rishyonot_krita/Documents/after_darom.XLS', - no after darom
+
+];
+
+const SHEET_BEFORE = 'Data2ToExcel_BeforDate';
+const SHEET_AFTER = 'Data2ToExcel_ToDate';
+const TIMEOUT_MS = 5000;
+
+const bucketName = Config.get('aws.treeBucketName');
```

Comment:
require both const at the same line 
const {bucketName, use S3} = config.get('aws')